aws/api_gateway/waf.tf

```diff
@@ -279,4 +279,89 @@ resource "aws_wafv2_regex_pattern_set" "anything" {
   regular_expression {
     regex_string = ".+"
   }
-}
\ No newline at end of file
+}
+
+#
+# Write WAF logs to S3
+#
+resource "aws_wafv2_web_acl_logging_configuration" "covid_metrics_waf_logs" {
+  log_destination_configs = [aws_kinesis_firehose_delivery_stream.covid_metrics_waf_logs.arn]
+  resource_arn            = aws_wafv2_web_acl.metrics_collection.arn
+}
+
+resource "aws_kinesis_firehose_delivery_stream" "covid_metrics_waf_logs" {
+  name        = "aws-waf-logs-covid-metrics"
+  destination = "extended_s3"
+
+  server_side_encryption {
+    enabled = true
+  }
+
+  extended_s3_configuration {
```

Comment:
Wondering if we need to change the default [buffer_size and buffer_interval](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kinesis_firehose_delivery_stream#buffer_size) due to the volume of requests this service gets.

Reply:
Good idea - lets chat about it.  There's also the Kinesis + S3 storage cost for this that may end up being prohibitive.

Reply:
Decided to remove WAF ACL logging based on cost.